src/components/Table/TableHeaders/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -167,7 +167,7 @@ exports[`components/page/Header renders routine headers correctly 1`] = `
             >
               <LinkAsButton
                 classNameProp="focus-investigation btn btn-primary float-right mt-5"
-                text="Add Focus Investigation"
+                text="Add Plan"
```

Comment:
@cKellyDesign I have asked about this change elsewhere so this comment is just to flag it for me.